src/freertos_drivers/common/WifiDefs.hxx

```diff
@@ -45,6 +45,12 @@ enum class CountryCode : uint8_t
     UNKNOWN, ///< unknown country code
 };
 
+enum WlanConnectResult
```

Comment:
Since this is at the toplevel namespace, please either
- make this an `enum class` (like the above enums)
- prefix all enum values with WLAN_CONNECT_RESULT_

because right now we have a symbol called 'OK' poisoning the toplevel namespace which will cause problems.